### ISSUE TITLE
Installation wizard - Channels cofiguration test buttons

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#198]
### User Story

As a node operator, I want to be able to verify that the channels I set up during installation, are correctly configured before my settings are finalised.

### Description

The scope of this task is **limited exclusively to the frontend aspect** of test buttons associated with the channels setup step - see #47.

### Requirements

A distinct 'Test' button is required any of the following alerting channels:

- Telegram
- Slack
- Email
- Opsgenie
- Pagerduty
- Twilio

Note that this requirement is independent of base-chain.

### Blocked by

Not dependent on another ticket to be tackled.

### Acceptance criteria

**Scenario**: Node operator is using the installation wizard to setup alerting channels
**Given**: The node operator is setting up alerting channels
**Then**: The node operator is presented with a Test button when setting up Telegram, Slack, Email, Opsgenie, Pagerduty, and Twilio alerting channels
